src/OpenLoco/Objects/ObjectManager.cpp

```diff
@@ -569,6 +574,34 @@ namespace OpenLoco::ObjectManager
         return checksum == object.checksum;
     }
 
+    static bool partialLoad(const ObjectHeader& header, stdx::span<uint8_t> objectData)
+    {
+        auto type = header.getType();
+        size_t index = 0;
+        for (; index < getMaxObjects(type); ++index)
+        {
+            if (object_repository[static_cast<uint8_t>(type)].objects[index] == reinterpret_cast<object*>(-1))
```

Comment:
We use this `static_cast<uint8_t>(type)` three times in the function. Worth saving it into a local const `repoOffset` or some such?